src/Ole32/Ole32.cs

```diff
@@ -0,0 +1,82 @@
+        ///     <listheader>
+        ///         <term>Return code</term>
+        ///         <term>Description</term>
+        ///     </listheader>
+        ///     <item>
+        ///         <term><c><see cref="HResult.Code.S_OK"/></c></term>
+        ///         <term>Call cancellation was successfully disabled on the thread.</term>
+        ///     </item>
+        ///     <item>
+        ///         <term><c><see cref="HResult.Code.CO_E_CANCEL_DISABLED"/></c></term>
+        ///         <term>There have been more successful calls to CoEnableCallCancellation on the thread than there have been calls to CoDisableCallCancellation. Cancellation is still enabled on the thread.</term>
+        ///     </item>
+        /// </list>
+        /// </returns>
+        [DllImport(nameof(Ole32))]
+        public static extern HResult CoDisableCallCancellation(IntPtr pReserved);
+
+        /// <summary>
+        /// Requests cancellation of an outbound DCOM method call pending on a specified thread.
+        /// </summary>
+        /// <param name="dwThreadId">The identifier of the thread on which the pending DCOM call is to be canceled. If this parameter is 0, the call is on the current thread.</param>
+        /// <param name="ulTimeout">The number of seconds <see cref="CoCancelCall"/> waits for the server to complete the outbound call after the client requests cancellation.</param>
+        /// <returns>This function can return the standard return values E_FAIL, E_OUTOFMEMORY, and E_UNEXPECTED, as well as the following values.
+        /// <list type="table">
+        ///     <listheader>
+        ///         <term>Return code</term>
+        ///         <term>Description</term>
+        ///     </listheader>
+        ///     <item>
+        ///         <term><c><see cref="HResult.Code.S_OK"/></c></term>
+        ///         <term>The cancellation request was made.</term>
+        ///     </item>
+        ///     <item>
+        ///         <term><c><see cref="HResult.Code.E_NOINTERFACE"/></c></term>
+        ///         <term><c>There is no cancel object corresponding to the specified thread.</c></term>
+        ///     </item>
+        ///     <item>
+        ///         <term><c><see cref="HResult.Code.CO_E_CANCEL_DISABLED"/></c></term>
+        ///         <term>Call cancellation is not enabled on the specified thread.</term>
+        ///     </item>
+        ///     <item>
+        ///         <term><c><see cref="HResult.Code.RPC_E_CALL_COMPLETE"/></c></term>
+        ///         <term>The call was completed during the timeout interval.</term>
+        ///     </item>
+        ///     <item>
+        ///         <term><c><see cref="HResult.Code.RPC_E_CALL_CANCELED"/></c></term>
+        ///         <term>The call was already canceled.</term>
+        ///     </item>
+        /// </list></returns>
+        [DllImport(nameof(Ole32))]
+        public static extern HResult CoCancelCall(uint dwThreadId, uint ulTimeout);
```

Comment:
We generally prefer to use `int` instead of `uint`, especially when it's likely that the user will want to pass in a variable that came from .NET. For example, .NET uses `int` for thread IDs, and if we use `uint` here it will require a cast. I would tend to say the same about your `ulTimeout` parameter. I suggest we make them both `int`.